libs/experimental/pyproject.toml

```diff
@@ -14,6 +14,9 @@ langchain = ">=0.0.239"
 presidio-anonymizer = {version = "^2.2.33", optional = true}
 presidio-analyzer = {version = "^2.2.33", optional = true}
 faker = {version = "^19.3.1", optional = true}
+vowpal-wabbit-next = {version = "0.6.0", optional = true}
+sentence-transformers = {version = "^2", optional = true}
+pandas = {version = "^2.0.1", optional = true}
```

Comment:
is pandas needed for unit tests? if not could we drop it here. think any dep that's not in an extras group will be default installed, even if it's optional.

Reply:
@baskaryan it isn't needed for the unit tests but it is needed for the chain to work. If I remove the pandas dep from here and re-build/install langchain_experimental, then the notebook I have breaks, so not sure I understand where pandas will become installed from

Reply:
@baskaryan tagging again :)

Reply:
@baskaryan I guess I'll tag once again :)

Reply:
user would be expected to manually install themselves, same as with all optional dependencies